plugins/com.aptana.git.ui/src/com/aptana/git/ui/internal/history/GitCompareFileRevisionEditorInput.java

```diff
@@ -425,6 +427,30 @@ else if (Utils.getAdapter(storage, IFile.class) != null)
 		return Messages.GitCompareFileRevisionEditorInput_2;
 	}
 
+	/**
+	 * This implementation was present within org.eclipse.team.internal.ui.Utils
+	 * until eclipse 3.7. But has been removed now and the implentation needed to be
+	 * repeated here.
+	 *  
+	 * @param storage
+	 * @param adapterType
+	 * @return
+	 */
+	private Object getAdapter(Object element, Class adapterType)
```

Comment:
I think we can remove this implementation

Reply:
Done!